storage/range.go

```diff
@@ -494,14 +494,8 @@ func (r *Range) SetLastVerificationTimestamp(timestamp proto.Timestamp) error {
 // either along the read-only execution path or the read-write Raft
 // command queue. If wait is false, read-write commands are added to
 // Raft without waiting for their completion.
-func (r *Range) AddCmd(ctx context.Context, call proto.Call, wait bool) error {
+func (r *Range) AddCmd(ctx context.Context, call proto.Call) error {
 	args, reply := call.Args, call.Reply
-	header := args.Header()
-	if !r.ContainsKeyRange(header.Key, header.EndKey) {
```

Comment:
Thanks for pointing out the post-raft thing; I've DRY'ed that using the new method I had to add here